crosschain-resolver/contracts/DelegatableResolverRegistrar.sol

```diff
@@ -0,0 +1,31 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4;
+import {DelegatableResolver} from "@ensdomains/ens-contracts/contracts/resolvers/DelegatableResolver.sol";
+
+/**
+ * A sample delegated resolver registrar that allows anyone to register subname
+ */
+contract DelegatableResolverRegistrar {
+    DelegatableResolver public resolver;
+
+    constructor(DelegatableResolver _resolver) {
+        resolver = _resolver;
+    }
+
+    /**
+     * @dev Approve an operator to be able to updated records on a node.
+     * @param name      The encoded subname
+     * @param operator  The address to approve
+     */
+
+    function register(bytes memory name, address operator) external {
+        (bytes32 node, bool authorized) = resolver.getAuthorisedNode(
```

Comment:
As I mentioned in the other PR, this will allow multiple people to register the same name!

Reply:
Please see my comment here https://github.com/ensdomains/ens-contracts/pull/295#discussion_r1421818550

Reply:
I don't understand the point of this function, then. What's the point in a contract that lets anyone claim any subdomain, regardless of whether it's already claimed?

Reply:
That's why I mentioned that it's just for the demo purpose. I can get rid of this and we can only add registrar when  we get to the phase 3 for the tokenised resolver.

Reply:
I think that's the best idea. Right now I think it's actively dangerous, as someone might deploy it under the mistaken assumption that it will give out domains on an exclusive basis.